experimental/generator-dotnet-yeoman/generators/app/templates/webapp/Dialogs/echo/echo.dialog

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
All of this is presumably the same as in the Functions project? Since the Dialogs directory and its contents should be identical between each, we should just have it the once and copy it to respective location within the generated project.